roles/download/tasks/download_container.yml

```diff
@@ -4,13 +4,16 @@
   when:
     - download.enabled
     - download.container
+    - not skip_downloads | default(false)
```

Comment:
The task that calls this, "download | Download files/images" already tests this. There's no value in adding this here. Honestly, it should actually not run any of these 3 tests because it's covered already. More tests = slower processing.

Reply:
This is because skip_downloads has been moved into the download files, because if you use skip_downlods then kubespray goes boomerang, because kublet, kubeadm etc. is not downloaded at all. So this fixes so you can actually use skip_downloads.